package.json

```diff
@@ -65,7 +78,7 @@
     "storybook-addon-designs": "^6.3.1",
     "style-dictionary": "^3.7.2",
     "styled-components": "5.3.8",
-    "typescript": "^4.9.5",
+    "typescript": "*",
```

Comment:
Minor issue; depending on`*` is not generally a good idea and could cause issues on npm install / upgrade